test/controllers/auth_controller_test.exs

```diff
@@ -3,7 +3,9 @@ defmodule AuthControllerTest do
   alias Constable.{User, UserIdentifier, UserInterest}
 
   @google_authorize_url "https://accounts.google.com/o/oauth2/auth"
-  @oauth_email_address "fake@thoughtbot.com"
+  @permitted_email_domain Application.fetch_env!(:constable, :permitted_email_domain)
+ 
+  def valid_email_address, do: "fake@#{@permitted_email_domain}"
```

Comment:
Is there a difference between the function definition here vs the variable definition in `user_test` (valid_email "foo@#{@permitted_email_domain}"`)? Is one preferable over the other?